src/main/java/org/usfirst/frc4048/common/swervev3/io/drive/SparkMaxDriveMotorIO.java

```diff
@@ -0,0 +1,49 @@
+package org.usfirst.frc4048.common.swervev3.io.drive;
+
+import com.revrobotics.CANSparkMax;
+import org.usfirst.frc4048.common.Constants;
+import org.usfirst.frc4048.common.swervev3.KinematicsConversionConfig;
+
+public class SparkMaxDriveMotorIO implements SwerveDriveMotorIO {
```

Comment:
For future consideration: This implementation somewhat betrays the notion of "no logic in the IO classes". I believe that if the logic (`setConversionFactors` etc) got moved into the `SwerveModule` then there would be no difference between swerve drive and steer motors.